Userland/Applications/Spreadsheet/Spreadsheet.cpp

```diff
@@ -98,20 +98,20 @@ static Optional<size_t> convert_from_string(StringView str, unsigned base = 26,
 
     VERIFY(base >= 2 && base <= map.length());
 
+    if (str.is_empty())
+        return {};
+
     size_t value = 0;
     auto const len = str.length();
     for (auto i = 0u; i < len; i++) {
         auto maybe_index = map.find(str[i]);
         if (!maybe_index.has_value())
             return {};
         size_t digit_value = maybe_index.value();
-        // NOTE: Refer to the note in `String::bijective_base_from()'.
-        if (i == 0 && len > 1)
-            ++digit_value;
-        value += digit_value * AK::pow<float>(base, len - 1 - i);
+        value += (digit_value + 1) * AK::pow<float>(base, len - 1 - i);
```

Comment:
itʼd be nice to have a test for that too…

as far as i understood it should be enough to assert that composing `convert_from_string` with `bijective_base_from` on both sides (for few relevant cases) is an identity map… (as we probably care more for those to be inverses than anything else)  🤔